### ISSUE TITLE
Videos glitch when in fullscreen (Youtube ++)

### ISSUE THREAD

**Issue by JohnnyMcNuggets** [eni9889/YT-PP-Issues#6]
When I am watching a video in widescreen mode for Youtube ++, It glitches and shows only the top half of the video. The only tweak for youtube I have is this one and it is updated to the lastest version. I don't know how to upload an image. I have screenshots if it.

**Comment by JohnnyMcNuggets** [comment]
IOS: 9.3.3
Youtube Version: Latest
Youtube ++ version: Latest

This all started when I tweeted from the app to get no ads.

**Comment by JMccormick264** [comment]
Does this happen every time?

**Comment by JohnnyMcNuggets** [comment]
@JMccormick264

Yep, it is happening all the time. I uninstalled it and checked if the the tweak was the problem and still is happening.

**Comment by JMccormick264** [comment]
Can you screenshot me what you are seeing? I have a ad removal account that I bought. But I can't repro. Could you screenshot for me?

**Comment by eni9889** [comment]
@JohnnyMcNuggets so this happens even without the tweak?
